docs/releases/release-8-3.md

```diff
@@ -0,0 +1,417 @@
+
+:::note
+The feature is enabled for new installations but not enforced on upgraded pools. We strongly advise enabling it after the upgrade.
+
+Enable it with: `xe pool-enable-tls-verification`.
+:::
+
+:::warning
+If you don't enable it after an upgrade, you may encounter situations where you try to add a new, freshly installed host to a pool, and it fails due to `POOL_JOINING_HOST_TLS_VERIFICATION_MISMATCH`. This happens because the new host needs to verify the identity of its peers, but the pool doesn't support it yet. To resolve this, enable TLS verification on the pool.
+:::
+
+More about this feature can be found [in XenServer's documentation](https://docs.xenserver.com/en-us/xenserver/8/hosts-pools/certificate-verification).
+
+### OpenSSH, security, ciphers, and user configuration
+
+Accepted ciphers, keys and algorithms used to be defined in `/etc/sshd_config` and `/etc/ssh_config`. Whenever we wanted to update them, we had to choose between potentially overwriting user modifications to these files, and not updating them (actually we had a middle ground where we attempted to patch just the relevant lines, but success was not guaranteed).
+
+Now it's simpler:
+- We define the accepted ciphers, keys, and algorithms directly at OpenSSH build time.
+- As long as users do not define settings that take precedence, we use our built-in defaults. To update them, we just publish a new build.
+- Users who choose to modify their SSH configuration will need to maintain these settings themselves, as any updates provided by our Security Team through new builds will be overridden by user-defined settings.
+
+### Restricted use of port 80 [XS]
+
+To enhance security, it is now possible to close TCP port 80 on the management interface. By default, port 80 remains open for specific clients that may still require it, but all internal connections now utilize HTTPS. Previously, VM migration occurred over port 80, but this has been updated.
+
+To close port 80, run the following command:
+```
+xe pool-param-set uuid=<pool-uuid> https-only=true
+```
+
+### Improvements in the installer
+
+Two years ago, development around XenServer's (and XCP-ng's) installer [was opened](https://github.com/xenserver/host-installer), allowing us to introduce several improvements and fixes (while previously we could modify the installer, contributing upstream reduces the long-term maintenance burden of divergence between our version and theirs).
+
+However, despite good intentions on both sides, not every change we submitted was accepted, leading to some remaining divergences. Unfortunately, some improvements we developed couldn’t be included in XCP-ng 8.3 as they would have caused significant differences with the upstream base (keeping large patches up to date as the base code evolves is not sustainable).
+
+Key improvements made to the installer include:
+- `memtest86+` has been updated to version 6 with EFI support.
```

Comment:
While the first part mentions only host-installer, which was the opened one, memtest is related to the install ISO, whose production process was not.  Also mention `create-install-image`?

Reply:
Agreed. Can you propose a rewording?

Reply:
proposal:
* "..., not every change we submitted was accepted, and the XS image-creation process is still not open, ..."
* new paragraph:

> We redesigned the image-creation process to [use a brand new set of scripts](https://github.com/xcp-ng/create-install-image/) - here again, this effort which started for 8.3 was already put to good use last year with XCP-ng 8.2.1's latest refreshed installation ISO.  Users willing to customize the installation images also have a more light-weight option, with our [iso-remaster script](https://github.com/xcp-ng/xcp/tree/master/scripts/iso-remaster).  And those facing a missing hardware driver at installation time will also have the option of [producing a Driver Disk](https://github.com/xcp-ng/driver-disks) to complement a pristine installation image.

Reply:
Added, thanks!